homeassistant/components/vilfo/const.py

```diff
@@ -0,0 +1,38 @@
+"""Constants for the Vilfo Router integration."""
+from homeassistant.const import DEVICE_CLASS_TIMESTAMP
+
+DOMAIN = "vilfo"
+
+ATTR_API_DATA_FIELD = "api_data_field"
+ATTR_API_DATA_FIELD_LOAD = "load"
+ATTR_API_DATA_FIELD_BOOT_TIME = "boot_time"
+ATTR_DEVICE_CLASS = "device_class"
+ATTR_ICON = "icon"
+ATTR_LABEL = "label"
+ATTR_LOAD = "load"
+ATTR_UNIT = "unit"
+ATTR_BOOT_TIME = "boot_time"
+
+ROUTER_DEFAULT_HOST = "admin.vilfo.com"
+ROUTER_DEFAULT_MODEL = "Vilfo Router"
+ROUTER_DEFAULT_NAME = "Vilfo Router"
+ROUTER_MANUFACTURER = "Vilfo AB"
+
+UNIT_PERCENT = "%"
+UNIT_ISO8601 = "ISO8601"
+
+SENSOR_TYPES = {
+    "load": {
+        ATTR_LABEL: ATTR_LOAD.replace("_", " ").title(),
+        ATTR_UNIT: UNIT_PERCENT,
+        ATTR_ICON: "mdi:memory",
+        ATTR_API_DATA_FIELD: ATTR_API_DATA_FIELD_LOAD,
+    },
+    "boot_time": {
+        ATTR_LABEL: ATTR_BOOT_TIME.replace("_", " ").title(),
+        ATTR_UNIT: UNIT_ISO8601,
```

Comment:
The timestamp device class sensor is unit less.

Reply:
OK, I can remove that. I based it off the docs you linked to:

> If specifying a device class, your sensor entity will need to also return the correct unit of measurement.

In the table the "unit" for `timestamp` is `ISO8601` (https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes)

Reply:
@MartinHjelmare Let me know your thoughts on how we can make that documentation better (regarding the `timestamp` and its unit) and I'll try to update that in a developer documentation PR that I'm about to start within a day or so. I've been trying to gather the things I've come across when using the docs 😄

Reply:
Not sure how to make the table clear. Add a fourth column with comments and remove ISO.... from the unit column?